spyder/utils/tests/test_encoding.py

```diff
@@ -31,6 +32,8 @@ def test_is_text_file(tmpdir):
 def test_files_encodings(expected_encoding, text_file):
     with open(os.path.join(__location__, text_file), 'rb') as f:
         text = f.read()
+        if PY2:
+            text = str(text)
```

Comment:
Are we assuring this also in the Editor, i.e. that text read from files in Python 2 is converted to bytes before detecting its encoding?